iao.owl

```diff
@@ -6967,15 +6967,161 @@ Data were captured into EPI-DATA (version 3.1), cleaned and then exported to Sta
     <!-- http://purl.obolibrary.org/obo/IAO_0020000 -->
 
     <owl:Class rdf:about="http://purl.obolibrary.org/obo/IAO_0020000">
+        <owl:equivalentClass>
+            <owl:Class>
+                <owl:intersectionOf rdf:parseType="Collection">
+                    <owl:Restriction>
+                        <owl:onProperty rdf:resource="http://purl.obolibrary.org/obo/IAO_0000219"/>
+                        <owl:someValuesFrom rdf:resource="http://purl.obolibrary.org/obo/BFO_0000001"/>
```

Comment:
This doesn't resolve.

Reply:
I'm not sure what to use here.  I can't find a working resource for BFO 'entity.'  The link below also has the same unresolvable url.  Can you point me in the right direction?

https://www.ebi.ac.uk/ols/ontologies/bfo/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FBFO_0000001&lang=en&viewMode=All&siblings=false

Reply:
@cmungall I am not quite sure what you mean. 
I'd prefer to change the axioms as: equivalent:  'information content entity' and (is_specified_output_of some 'identifier creating process') and subClassOf: denotes some entity